config/config.go

```diff
@@ -1209,20 +1222,31 @@ func Unmarshal(yamlString string) (conf *Config, err error) {
 	if !conf.Deployment.ClusterWideAccess {
 		conf.Deployment.AccessibleNamespaces, err = conf.extractAccessibleNamespaceList()
 		if err != nil {
+			conf.Credentials.Close()
+			conf.Credentials = nil
 			return nil, err
 		}
 	}
 
 	conf.prepareDashboards()
 
-	additionalCABundles := []string{additionalCABundle}
 	// Auth strategy isn't a great proxy for whether the cluster is running on openshift or not
 	// but the config is the very first thing loaded so we don't have access to a client.
-	if conf.Auth.Strategy == AuthStrategyOpenshift {
-		additionalCABundles = append(additionalCABundles, openshiftServingCAFromSA, openshiftServingCA)
+	var additionalCABundles []string
```

Comment:
nit: adding the `additionalCABundle` here will mean you don't need to add it to each individual strategy since we always want to include it in `additionalCABundles`. It will also eliminate the `default` case.

Reply:
I purposely added the default case and created the empty var to make it explicit for readers of the code and to make it more easy to see what each strategy explicitly has in them. Otherwise, you will see the list and then in your mind have to go back up in the code and say, "oh , wait, but there is this other one that is defined earlier that is in here too"